examples/basic/main.tf

```diff
@@ -1,22 +1,23 @@
-###############################################################################
+########################################################################################################################
 # Resource Group
-###############################################################################
+########################################################################################################################
 
 module "resource_group" {
   source  = "terraform-ibm-modules/resource-group/ibm"
   version = "1.1.4"
-  # if an existing resource group is not set (i.e. null) create a new one using prefix
+  # if an existing resource group is not set (null) create a new one using prefix
   resource_group_name          = var.resource_group == null ? "${var.prefix}-resource-group" : null
   existing_resource_group_name = var.resource_group
 }
 
-##############################################################################
-# Create a VPC with single subnet and zone, and public gateway
-# NOTE: this is a very simple VPC/Subnet configuration for example purposes only,
-# that will allow all traffic ingress/egress by default.
-# For production use cases this would need to be enhanced by adding more subnets
-# and zones for resiliency, and ACLs/Security Groups for network security.
-##############################################################################
+########################################################################################################################
```

Comment:
We can remove this comment as all the information is provided in the example's readme

Reply:
Well this is more info than the readme. I purposely have it in here incase anyone goes to copy and paste the VPC part, the code comment will warn them its not for production. We need to add more code comments to our examples